Picked Local content provider.
Using local repo /tmp/r2d-test-2020-07-08T06.18af5q_72d/github.com/scipy/scipy.
Using PythonBuildPack builder
Step 1/48 : FROM buildpack-deps:bionic
 ---> 5d109dc6d8a0
Step 2/48 : ENV DEBIAN_FRONTEND=noninteractive
 ---> Using cache
 ---> 84875aa039ff
Step 3/48 : RUN apt-get -qq update &&     apt-get -qq install --yes --no-install-recommends locales > /dev/null &&     apt-get -qq purge &&     apt-get -qq clean &&     rm -rf /var/lib/apt/lists/*
 ---> Using cache
 ---> 2c6f3445209c
Step 4/48 : RUN echo "en_US.UTF-8 UTF-8" > /etc/locale.gen &&     locale-gen
 ---> Using cache
 ---> 5ac40a1069ef
Step 5/48 : ENV LC_ALL en_US.UTF-8
 ---> Using cache
 ---> 0569ecd6efb3
Step 6/48 : ENV LANG en_US.UTF-8
 ---> Using cache
 ---> bd3372a8f400
Step 7/48 : ENV LANGUAGE en_US.UTF-8
 ---> Using cache
 ---> ab87a637d837
Step 8/48 : ENV SHELL /bin/bash
 ---> Using cache
 ---> 50ed7363eaf5
Step 9/48 : ARG NB_USER
 ---> Using cache
 ---> 0b8893bd6422
Step 10/48 : ARG NB_UID
 ---> Using cache
 ---> b9e6fa4ea16e
Step 11/48 : ENV USER ${NB_USER}
 ---> Using cache
 ---> b142a4c01942
Step 12/48 : ENV HOME /home/${NB_USER}
 ---> Using cache
 ---> e11d5723ab90
Step 13/48 : RUN groupadd         --gid ${NB_UID}         ${NB_USER} &&     useradd         --comment "Default user"         --create-home         --gid ${NB_UID}         --no-log-init         --shell /bin/bash         --uid ${NB_UID}         ${NB_USER}
 ---> Using cache
 ---> e20a5903ae77
Step 14/48 : RUN wget --quiet -O - https://deb.nodesource.com/gpgkey/nodesource.gpg.key |  apt-key add - &&     DISTRO="bionic" &&     echo "deb https://deb.nodesource.com/node_10.x $DISTRO main" >> /etc/apt/sources.list.d/nodesource.list &&     echo "deb-src https://deb.nodesource.com/node_10.x $DISTRO main" >> /etc/apt/sources.list.d/nodesource.list
 ---> Using cache
 ---> 730bb79c819d
Step 15/48 : RUN apt-get -qq update &&     apt-get -qq install --yes --no-install-recommends        less        nodejs        unzip        > /dev/null &&     apt-get -qq purge &&     apt-get -qq clean &&     rm -rf /var/lib/apt/lists/*
 ---> Using cache
 ---> 2be5b9d2edc7
Step 16/48 : EXPOSE 8888
 ---> Using cache
 ---> 7fcdbaf0fe40
Step 17/48 : ENV APP_BASE /srv
 ---> Using cache
 ---> ee51ea8ec11a
Step 18/48 : ENV NPM_DIR ${APP_BASE}/npm
 ---> Using cache
 ---> f34fdbe18446
Step 19/48 : ENV NPM_CONFIG_GLOBALCONFIG ${NPM_DIR}/npmrc
 ---> Using cache
 ---> 833e25d9d235
Step 20/48 : ENV CONDA_DIR ${APP_BASE}/conda
 ---> Using cache
 ---> 8ee468f15616
Step 21/48 : ENV NB_PYTHON_PREFIX ${CONDA_DIR}/envs/notebook
 ---> Using cache
 ---> 971c9391c0ab
Step 22/48 : ENV KERNEL_PYTHON_PREFIX ${NB_PYTHON_PREFIX}
 ---> Using cache
 ---> 7fa50abe8e21
Step 23/48 : ENV PATH ${NB_PYTHON_PREFIX}/bin:${CONDA_DIR}/bin:${NPM_DIR}/bin:${PATH}
 ---> Using cache
 ---> eaa621b5d97f
Step 24/48 : COPY build_script_files/-2fusr-2flocal-2flib-2fpython3-2e8-2fdist-2dpackages-2frepo2docker-2fbuildpacks-2fconda-2factivate-2dconda-2esh-1be18e /etc/profile.d/activate-conda.sh
 ---> Using cache
 ---> 9ffab983afa0
Step 25/48 : COPY build_script_files/-2fusr-2flocal-2flib-2fpython3-2e8-2fdist-2dpackages-2frepo2docker-2fbuildpacks-2fconda-2fenvironment-2epy-2d3-2e7-2efrozen-2eyml-26bd38 /tmp/environment.yml
 ---> Using cache
 ---> 9b3e6c528ba6
Step 26/48 : COPY build_script_files/-2fusr-2flocal-2flib-2fpython3-2e8-2fdist-2dpackages-2frepo2docker-2fbuildpacks-2fconda-2finstall-2dminiconda-2ebash-813f10 /tmp/install-miniconda.bash
 ---> Using cache
 ---> e52d3b4016a9
Step 27/48 : RUN mkdir -p ${NPM_DIR} && chown -R ${NB_USER}:${NB_USER} ${NPM_DIR}
 ---> Using cache
 ---> 0a8f7bcc0bd7
Step 28/48 : USER ${NB_USER}
 ---> Using cache
 ---> e295fca18e9d
Step 29/48 : RUN npm config --global set prefix ${NPM_DIR}
 ---> Using cache
 ---> b3629329dd8f
Step 30/48 : USER root
 ---> Using cache
 ---> 5ba3698f0957
Step 31/48 : RUN bash /tmp/install-miniconda.bash && rm /tmp/install-miniconda.bash /tmp/environment.yml
 ---> Using cache
 ---> ecb18ec9bb2f
Step 32/48 : ARG REPO_DIR=${HOME}
 ---> Using cache
 ---> 8f78500beba3
Step 33/48 : ENV REPO_DIR ${REPO_DIR}
 ---> Using cache
 ---> 300bfe3eb940
Step 34/48 : WORKDIR ${REPO_DIR}
 ---> Using cache
 ---> ab56d4c94aaf
Step 35/48 : ENV PATH ${HOME}/.local/bin:${REPO_DIR}/.local/bin:${PATH}
 ---> Using cache
 ---> 26a826700420
Step 36/48 : ENV CONDA_DEFAULT_ENV ${KERNEL_PYTHON_PREFIX}
 ---> Using cache
 ---> ebb7f28a23e9
Step 37/48 : USER root
 ---> Using cache
 ---> 4477f562c4c4
Step 38/48 : COPY src/ ${REPO_DIR}
 ---> 37443674dd72
Step 39/48 : RUN chown -R ${NB_USER}:${NB_USER} ${REPO_DIR}
 ---> Running in 3b4a6ef5dc7d
Removing intermediate container 3b4a6ef5dc7d
 ---> 1e2e58498206
Step 40/48 : USER ${NB_USER}
 ---> Running in 12b5c8f88f6e
Removing intermediate container 12b5c8f88f6e
 ---> 845f2bb3762e
Step 41/48 : RUN ${KERNEL_PYTHON_PREFIX}/bin/pip install --no-cache-dir .
 ---> Running in 6fc4e6ce8cd7
[91mWARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.
To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.
[0mProcessing /home/vildeeide
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'done'
    Preparing wheel metadata: started
    Preparing wheel metadata: still running...
    Preparing wheel metadata: finished with status 'error'
[91m    ERROR: Command errored out with exit status 1:
     command: /srv/conda/envs/notebook/bin/python /srv/conda/envs/notebook/lib/python3.7/site-packages/pip/_vendor/pep517/_in_process.py prepare_metadata_for_build_wheel /tmp/tmpbaipqw9w
         cwd: /tmp/pip-req-build-62wl1luk
    Complete output (218 lines):
    Running scipy/linalg/_generate_pyx.py
    Running scipy/special/_generate_pyx.py
    Processing scipy/sparse/_csparsetools.pyx.in
    Processing scipy/sparse/csgraph/_min_spanning_tree.pyx
    Processing scipy/sparse/csgraph/_reordering.pyx
    Processing scipy/sparse/csgraph/_shortest_path.pyx
    Processing scipy/sparse/csgraph/_traversal.pyx
    Processing scipy/sparse/csgraph/_tools.pyx
    Processing scipy/sparse/csgraph/_matching.pyx
    Processing scipy/sparse/csgraph/_flow.pyx
    Processing scipy/signal/_peak_finding_utils.pyx
    Processing scipy/signal/_max_len_seq_inner.pyx
    Processing scipy/signal/_upfirdn_apply.pyx
    Processing scipy/signal/_spectral.pyx
    Processing scipy/signal/_sosfilt.pyx
    Processing scipy/spatial/qhull.pyx
    Processing scipy/spatial/_hausdorff.pyx
    Processing scipy/spatial/ckdtree.pyx
    Processing scipy/spatial/_voronoi.pyx
    Processing scipy/spatial/transform/rotation.pyx
    Processing scipy/stats/_stats.pyx
    Processing scipy/linalg/cython_blas.pyx
    Processing scipy/linalg/_solve_toeplitz.pyx
    Processing scipy/linalg/cython_lapack.pyx
    Processing scipy/linalg/_matfuncs_sqrtm_triu.pyx
    Processing scipy/linalg/_decomp_update.pyx.in
    Processing scipy/optimize/_bglu_dense.pyx
    Processing scipy/optimize/_group_columns.pyx
    Processing scipy/optimize/cython_optimize/_zeros.pyx.in
    Processing scipy/optimize/_lsq/givens_elimination.pyx
    Processing scipy/optimize/_trlib/_trlib.pyx
    Processing scipy/special/cython_special.pyx
    Processing scipy/special/_comb.pyx
    Processing scipy/special/_test_round.pyx
    Processing scipy/special/_ufuncs.pyx
    warning: _cython_special_custom.pxi:9:8: Unreachable code
    warning: _cython_special_custom.pxi:13:4: Unreachable code
    warning: _cython_special_custom.pxi:21:8: Unreachable code
    warning: _cython_special_custom.pxi:25:4: Unreachable code
    warning: _cython_special_custom.pxi:33:8: Unreachable code
    warning: _cython_special_custom.pxi:37:4: Unreachable code
    warning: _cython_special_custom.pxi:45:8: Unreachable code
    warning: _cython_special_custom.pxi:49:4: Unreachable code
    Processing scipy/special/_ufuncs_cxx.pyx
    Processing scipy/special/_ellip_harm_2.pyx
    Processing scipy/ndimage/src/_cytest.pyx
    Processing scipy/ndimage/src/_ni_label.pyx
    Processing scipy/io/matlab/mio5_utils.pyx
    Processing scipy/io/matlab/mio_utils.pyx
    Processing scipy/io/matlab/streams.pyx
    Processing scipy/interpolate/_bspl.pyx
    Processing scipy/interpolate/interpnd.pyx
    Processing scipy/interpolate/_ppoly.pyx
    Processing scipy/fftpack/convolve.pyx
    Processing scipy/_lib/_test_deprecation_call.pyx
    Processing scipy/_lib/messagestream.pyx
    Processing scipy/_lib/_test_deprecation_def.pyx
    Processing scipy/_lib/_ccallback_c.pyx
    Processing scipy/cluster/_hierarchy.pyx
    Processing scipy/cluster/_vq.pyx
    Processing scipy/cluster/_optimal_leaf_ordering.pyx
    Cythonizing sources
    lapack_opt_info:
    lapack_mkl_info:
    customize UnixCCompiler
      libraries mkl_rt not found in ['/srv/conda/envs/notebook/lib', '/usr/local/lib', '/usr/lib', '/usr/lib/x86_64-linux-gnu']
      NOT AVAILABLE
    
    openblas_lapack_info:
    customize UnixCCompiler
    customize UnixCCompiler
      libraries openblas not found in ['/srv/conda/envs/notebook/lib', '/usr/local/lib', '/usr/lib', '/usr/lib/x86_64-linux-gnu']
      NOT AVAILABLE
    
    openblas_clapack_info:
    customize UnixCCompiler
    customize UnixCCompiler
      libraries openblas,lapack not found in ['/srv/conda/envs/notebook/lib', '/usr/local/lib', '/usr/lib', '/usr/lib/x86_64-linux-gnu']
      NOT AVAILABLE
    
    atlas_3_10_threads_info:
    Setting PTATLAS=ATLAS
    customize UnixCCompiler
      libraries tatlas,tatlas not found in /srv/conda/envs/notebook/lib
    customize UnixCCompiler
      libraries lapack_atlas not found in /srv/conda/envs/notebook/lib
    customize UnixCCompiler
      libraries tatlas,tatlas not found in /usr/local/lib
    customize UnixCCompiler
      libraries lapack_atlas not found in /usr/local/lib
    customize UnixCCompiler
      libraries tatlas,tatlas not found in /usr/lib
    customize UnixCCompiler
      libraries lapack_atlas not found in /usr/lib
    customize UnixCCompiler
      libraries tatlas,tatlas not found in /usr/lib/x86_64-linux-gnu
    customize UnixCCompiler
      libraries lapack_atlas not found in /usr/lib/x86_64-linux-gnu
    <class 'numpy.distutils.system_info.atlas_3_10_threads_info'>
      NOT AVAILABLE
    
    atlas_3_10_info:
    customize UnixCCompiler
      libraries satlas,satlas not found in /srv/conda/envs/notebook/lib
    customize UnixCCompiler
      libraries lapack_atlas not found in /srv/conda/envs/notebook/lib
    customize UnixCCompiler
      libraries satlas,satlas not found in /usr/local/lib
    customize UnixCCompiler
      libraries lapack_atlas not found in /usr/local/lib
    customize UnixCCompiler
      libraries satlas,satlas not found in /usr/lib
    customize UnixCCompiler
      libraries lapack_atlas not found in /usr/lib
    customize UnixCCompiler
      libraries satlas,satlas not found in /usr/lib/x86_64-linux-gnu
    customize UnixCCompiler
      libraries lapack_atlas not found in /usr/lib/x86_64-linux-gnu
    <class 'numpy.distutils.system_info.atlas_3_10_info'>
      NOT AVAILABLE
    
    atlas_threads_info:
    Setting PTATLAS=ATLAS
    customize UnixCCompiler
      libraries ptf77blas,ptcblas,atlas not found in /srv/conda/envs/notebook/lib
    customize UnixCCompiler
      libraries lapack_atlas not found in /srv/conda/envs/notebook/lib
    customize UnixCCompiler
      libraries ptf77blas,ptcblas,atlas not found in /usr/local/lib
    customize UnixCCompiler
      libraries lapack_atlas not found in /usr/local/lib
    customize UnixCCompiler
      libraries ptf77blas,ptcblas,atlas not found in /usr/lib
    customize UnixCCompiler
      libraries lapack_atlas not found in /usr/lib
    customize UnixCCompiler
      libraries ptf77blas,ptcblas,atlas not found in /usr/lib/x86_64-linux-gnu
    customize UnixCCompiler
      libraries lapack_atlas not found in /usr/lib/x86_64-linux-gnu
    <class 'numpy.distutils.system_info.atlas_threads_info'>
      NOT AVAILABLE
    
    atlas_info:
    customize UnixCCompiler
      libraries f77blas,cblas,atlas not found in /srv/conda/envs/notebook/lib
    customize UnixCCompiler
      libraries lapack_atlas not found in /srv/conda/envs/notebook/lib
    customize UnixCCompiler
      libraries f77blas,cblas,atlas not found in /usr/local/lib
    customize UnixCCompiler
      libraries lapack_atlas not found in /usr/local/lib
    customize UnixCCompiler
      libraries f77blas,cblas,atlas not found in /usr/lib
    customize UnixCCompiler
      libraries lapack_atlas not found in /usr/lib
    customize UnixCCompiler
      libraries f77blas,cblas,atlas not found in /usr/lib/x86_64-linux-gnu
    customize UnixCCompiler
      libraries lapack_atlas not found in /usr/lib/x86_64-linux-gnu
    <class 'numpy.distutils.system_info.atlas_info'>
      NOT AVAILABLE
    
    lapack_info:
    customize UnixCCompiler
      libraries lapack not found in ['/srv/conda/envs/notebook/lib', '/usr/local/lib', '/usr/lib', '/usr/lib/x86_64-linux-gnu']
      NOT AVAILABLE
    
    lapack_src_info:
      NOT AVAILABLE
    
      NOT AVAILABLE
    
    setup.py:462: UserWarning: Unrecognized setuptools command ('dist_info --egg-base /tmp/pip-modern-metadata-akmtfb5o'), proceeding with generating Cython sources and expanding templates
      ' '.join(sys.argv[1:])))
    Running from SciPy source directory.
    /tmp/pip-build-env-njftkhr_/overlay/lib/python3.7/site-packages/numpy/distutils/system_info.py:624: UserWarning:
        Atlas (http://math-atlas.sourceforge.net/) libraries not found.
        Directories to search for the libraries can be specified in the
        numpy/distutils/site.cfg file (section [atlas]) or by setting
        the ATLAS environment variable.
      self.calc_info()
    /tmp/pip-build-env-njftkhr_/overlay/lib/python3.7/site-packages/numpy/distutils/system_info.py:624: UserWarning:
        Lapack (http://www.netlib.org/lapack/) libraries not found.
        Directories to search for the libraries can be specified in the
        numpy/distutils/site.cfg file (section [lapack]) or by setting
        the LAPACK environment variable.
      self.calc_info()
    /tmp/pip-build-env-njftkhr_/overlay/lib/python3.7/site-packages/numpy/distutils/system_info.py:624: UserWarning:
        Lapack (http://www.netlib.org/lapack/) sources not found.
        Directories to search for the sources can be specified in the
        numpy/distutils/site.cfg file (section [lapack_src]) or by setting
        the LAPACK_SRC environment variable.
      self.calc_info()
    Traceback (most recent call last):
      File "/srv/conda/envs/notebook/lib/python3.7/site-packages/pip/_vendor/pep517/_in_process.py", line 257, in <module>
        main()
      File "/srv/conda/envs/notebook/lib/python3.7/site-packages/pip/_vendor/pep517/_in_process.py", line 240, in main
        json_out['return_val'] = hook(**hook_input['kwargs'])
      File "/srv/conda/envs/notebook/lib/python3.7/site-packages/pip/_vendor/pep517/_in_process.py", line 110, in prepare_metadata_for_build_wheel
        return hook(metadata_directory, config_settings)
      File "/tmp/pip-build-env-njftkhr_/overlay/lib/python3.7/site-packages/setuptools/build_meta.py", line 157, in prepare_metadata_for_build_wheel
        self.run_setup()
      File "/tmp/pip-build-env-njftkhr_/overlay/lib/python3.7/site-packages/setuptools/build_meta.py", line 249, in run_setup
        self).run_setup(setup_script=setup_script)
      File "/tmp/pip-build-env-njftkhr_/overlay/lib/python3.7/site-packages/setuptools/build_meta.py", line 142, in run_setup
        exec(compile(code, __file__, 'exec'), locals())
      File "setup.py", line 590, in <module>
        setup_package()
      File "setup.py", line 586, in setup_package
        setup(**metadata)
      File "/tmp/pip-build-env-njftkhr_/overlay/lib/python3.7/site-packages/numpy/distutils/core.py", line 135, in setup
        config = configuration()
      File "setup.py", line 484, in configuration
        raise NotFoundError(msg)
    numpy.distutils.system_info.NotFoundError: No BLAS/LAPACK libraries found.
    To build Scipy from sources, BLAS & LAPACK libraries need to be installed.
    See site.cfg.example in the Scipy source directory and
    https://docs.scipy.org/doc/scipy/reference/building/index.html for details.
    ----------------------------------------
[0m[91mERROR: Command errored out with exit status 1: /srv/conda/envs/notebook/bin/python /srv/conda/envs/notebook/lib/python3.7/site-packages/pip/_vendor/pep517/_in_process.py prepare_metadata_for_build_wheel /tmp/tmpbaipqw9w Check the logs for full command output.
[0mRemoving intermediate container 6fc4e6ce8cd7
The command '/bin/sh -c ${KERNEL_PYTHON_PREFIX}/bin/pip install --no-cache-dir .' returned a non-zero code: 1Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/repo2docker_checker/checker.py", line 318, in test_one_repo
    image, checkout_path = build_repo(
  File "/usr/local/lib/python3.8/dist-packages/repo2docker_checker/checker.py", line 128, in build_repo
    run(
  File "/usr/lib/python3.8/subprocess.py", line 512, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['jupyter-repo2docker', '--no-run', '--no-clean', '--image-name', 'r2d-test-github.com-scipy-scipy:289ae46', '/tmp/r2d-test-2020-07-08T06.18af5q_72d/github.com/scipy/scipy']' returned non-zero exit status 1.
